jwst/associations/lib/rules_level2b.py

```diff
@@ -597,6 +597,12 @@ def add_catalog_members(self):
                 'exptype': 'sourcecat'
             })
         )
+        members.append(
+            Member({
+                'expname': lv3_direct_image_root + '_seg.fits',
```

Comment:
The PR that updated the `source_cat` step used a suffix of "segm", not "seg"

Reply:
😬